test/helper-functions.bats

```diff
@@ -0,0 +1,43 @@
+load 'test_helper/common'
+
+function setup() {
+    run_setup_file_if_necessary
+}
+
+function teardown() {
+    run_teardown_file_if_necessary
+}
+
+function setup_file() {
+    local PRIVATE_CONFIG
+    PRIVATE_CONFIG="$(duplicate_config_for_container .)"
+    docker run -d --name mail_helper_functions \
+		-v "${PRIVATE_CONFIG}":/tmp/docker-mailserver \
+		-v "$(pwd)/test/test-files":/tmp/docker-mailserver-test:ro \
+		-e ENABLE_FETCHMAIL=1 \
+		--cap-add=NET_ADMIN \
+		-e DMS_DEBUG=0 \
+		-h mail.my-domain.com -t "${NAME}"
+    wait_for_finished_setup_in_container mail_helper_functions
+}
+
+function teardown_file() {
+    docker rm -f mail_helper_functions
+}
+
+@test "first" {
+  skip 'this test must come first to reliably identify when to run setup_file'
+}
```

Comment:
Match same indentation of level of 4 spaces per tab in this file.

Yes I'm aware the other bats file in this PR is using 2 spaces instead, if someone cares to normalize that via a lint tool later great, but for now I only care about file level consistency :)

```suggestion
@test "first" {
    skip 'this test must come first to reliably identify when to run setup_file'
}
```